docker-compose.yml

```diff
@@ -29,5 +29,17 @@ services:
       - SPRING_PROFILES_ACTIVE=dev
       - APPLICATION_AUTHENTICATION_UI_ALLOWLIST=0.0.0.0/0
 
+  db:
```

Comment:
Not strictly related to creating the swagger spec, but added a DB to the docker-compose so that we can run the app locally